client/server/middleware/unsupported-browser.js

```diff
@@ -18,13 +18,32 @@ function isSupportedBrowser( req ) {
 	} );
 }
 
+// We don't want to redirect some of our public landing pages, so we include them
+// here.
+function allowPath( path ) {
+	// Strip leading '/'.
```

Comment:
The leading slash is always guaranteed to be there, so we don't need to assume it's optional.

Reply:
I wrote it this way to avoid prefixing `/` to all the paths and locales we try to match :)

Reply:
Prefixing all the paths is fine, we use it all the time e.g. in `client/layout/index.jsx`. And in theory the `/log-in` string should be easier to grep than `log-in`, like when answering a "what are all the places where we work with the `/log-in` route?" question.

Reply:
Hm, yeah that's a good point. I'll follow up on that today

Reply:
#56765